tests/e2e/tests/pilot/testdata/authn/v1alpha1/authn-policy.yaml.tmpl

```diff
@@ -6,5 +6,5 @@ spec:
   targets:
   - name: "c"
   - name: "d"
-    ports:
-    - number: 80
+  peers:
+  - mtls: {}
```

Comment:
The name of this rule indicates 'disable mtls'. But is it actually for enabling mTLS for 'c' and 'd'?

Reply:
+1. Name should be updated to be consistent with the new intention. Another reason for not changing the test set up.

Reply:
@diemtvu , i can restore the existing policy setup for this case, but there are many test cases we do want to update, check the failing tests. https://circleci.com/gh/istio/istio/155866?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
It's probably a true problem. One of the failure test is  TestGateway_TCPIngress/TCPIngressGateway. Do you think PERMISSIVE could do smt wrong for TCP?